src/bridge/RoomAccessSyncer.ts

```diff
@@ -310,9 +318,18 @@ export class RoomAccessSyncer {
                 // set of modes.
                 level = 0;
             }
-            this.setPowerLevel(room.getId(), userId, level, req);
+            try {
+                req.log.info(`Granting PL${level} to ${userId} in ${roomId}`);
+                await this.setPowerLevel(roomId, userId, level, req);
+            }
+            catch (ex) {
+                req.log.warn(`Failed to grant PL in ${roomId}`, ex);
+                didFail = true;
+            }
+        }
+        if (didFail) {
+            throw new Error('Failed to update PL in soime rooms');
```

Comment:
```suggestion
            throw new Error('Failed to update PL in some rooms');
```
...to ease log searching.